.github/configs/labeler.yml

```diff
@@ -0,0 +1,130 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build:
+  - changed-files:
+    - any-glob-to-any-file:
+      - '.github/**'
+      - 'gradle/**'
+      - '*gradlew*'
+
+clients:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'clients/**'
+
+
+tests:
+- all:
+  - changed-files:
+    - any-glob-to-all-files: 
+      - 'tests/**'
+      - '**/test/**'
+
+
+connect:
+- all:
+  - changed-files:
+    - any-glob-to-any-file: 
+      - 'connect/**/*'
+    - all-globs-to-all-files:
+      - '!connect/mirror/*'
+
+mirror-maker-2:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'connect/mirror/**'
+
+
+core:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'core/**'
+    - '*server*/**'
+
+storage:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'storage/**'
+
+streams:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'streams/**'
+
+tools:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'tools/**'
+    - 'trogdor/**'
+    - 'vagrant/**'
+    - 'committer-tools/**'
+    - 'log4j-appender/**'
+    - 'shell/**'
+
+docs:
+- changed-files:
+  - any-glob-to-all-files:
+    - 'docs/**'
+    - '**/*.md'
+    - 'examples/**'
+
+generator:
+  - changed-files:
+      - any-glob-to-any-file:
+        - 'generator/**'
+
+transaction:
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'transaction*/**'
+
+share:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '*share*/**'
+
+docker:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'docker/**'
+
+
+coordinator:
```

Comment:
This is not a label. Let's remove this one. We can let the developers working on these areas propose their own globs and labels